_includes/2017_data/sponsors.html

```diff
@@ -12,6 +12,14 @@ <h2 class="section-heading" id="sponsors">Sponsors</h2>
                     <h3 class="section-heading">Metropolis-Hastings Tier</h3>
```

Comment:
Would add another section below the tidelift image for "technology partners" and then move the added code below